docker/main/rootfs/etc/s6-overlay/s6-rc.d/go2rtc/run

```diff
@@ -45,15 +45,20 @@ function get_ip_and_port_from_supervisor() {
 
 export LIBAVFORMAT_VERSION_MAJOR=$(ffmpeg -version | grep -Po 'libavformat\W+\K\d+')
 
+if [[ -f "/dev/shm/go2rtc.yaml" ]]; then
+    echo "[INFO] Removing stale config from last run..."
+    rm /dev/shm/go2rtc.yaml
+
 if [[ ! -f "/dev/shm/go2rtc.yaml" ]]; then
-    echo "[INFO] Preparing go2rtc config..."
+    echo "[INFO] Preparing new go2rtc config..."
+    python3 /usr/local/go2rtc/create_config.py
```

Comment:
this needs to be run after the supervisor logic below

Reply:
I have nothing in my config specifying /dev/shm

    volumes:
      - /dev:/dev
      - ./config/frigate.yml:/config/config.yml:ro
      - ./data:/media/frigate
      - type: tmpfs
        target: /tmp/cache
        tmpfs:
          size: 1000000000

Reply:
> this needs to be run after the supervisor logic below

Yeah I don't know anything about the home assistant integration, I run it through the proxy addon.

Reply:
you are mapping /dev/ which of course includes /dev/shm

Reply:
> > this needs to be run after the supervisor logic below
> 
> Yeah I don't know anything about the home assistant integration, I run it through the proxy addon.

right, in those cases it is best to leave the code as is and not move it around unless you know what you are changing

Reply:
> you are mapping /dev/ which of course includes /dev/shm

bet ... that was a very old troubleshooting move I made while working out nvidia decoding.